ext/standard/php_dir.h

```diff
@@ -17,12 +17,59 @@
 #ifndef PHP_DIR_H
 #define PHP_DIR_H
 
-/* directory functions */
-PHP_MINIT_FUNCTION(dir);
-PHP_RINIT_FUNCTION(dir);
+#ifdef HAVE_GLOB
```

Comment:
Thinking a bit more about this... I'm not exactly sure it's a good idea to put these includes here. I mean, the constants below need them for sure, but this file is also required by `php_standard.h`, which would mean that the (unused) includes will be available for it as well.

That's why it seems to me that my suggestion of putting the remaining `#define`s here was not completely right, because most probably a newly added, internal header file should be used for this purpose. We have a few similar examples, like `php_pdo_pgsql_int.h` and some other headers which have `_int`, `_internal` suffix. @nielsdos Could you please share your thoughts here?

Reply:
Whoops, I don't know how but I somehow missed your question, sorry.
I agree with the _int or _internal idea, as seems to be done now :-)